javatests/arcs/core/storage/BackingStoreTest.kt

```diff
@@ -0,0 +1,102 @@
+package arcs.core.storage
+
+import arcs.core.crdt.CrdtEntity
+import arcs.core.crdt.CrdtSingleton
+import arcs.core.crdt.VersionMap
+import arcs.core.data.EntityType
+import arcs.core.data.FieldType
+import arcs.core.data.Schema
+import arcs.core.data.SchemaFields
+import arcs.core.data.util.toReferencable
+import arcs.core.storage.api.DriverAndKeyConfigurator
+import arcs.core.storage.keys.RamDiskStorageKey
+import com.google.common.truth.Truth.assertThat
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.coroutineScope
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.runBlocking
+import org.junit.Test
+
+class BackingStoreTest {
```

Comment:
```suggestion
import org.junit.Test
import org.junit.runner.RunWith
import org.junit.runners.JUnit4

@RunWith(JUnit4::class)
class BackingStoreTest {
```

 (not 100% sure on the imports here)